superset/assets/javascripts/explore/components/MetricDefinitionOption.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import MetricOption from '../../components/MetricOption';
+import ColumnOption from '../../components/ColumnOption';
+import AggregateOption from './AggregateOption';
+
+const propTypes = {
+  option: PropTypes.object.isRequired,
```

Comment:
can you define a re-usable `option` shape? typically `PropTypes.object` is looked down upon because it could be literally anything and this seems like it's used in a few places

Reply:
yeah, I can clean this one up